onset_maproom/maproom.py

```diff
@@ -58,21 +59,38 @@ def toggle_navbar_collapse(n, is_open):
     return is_open
 
 
-#@APP.callback(
-#    Output("probability-collapse", "is_open"),
-#    Output("probExcThresh1", "value"),
-#    Output("poeunits", "value"),
-#    Input("yearly_stats_input", "value"),
-#)
-#def change_form_layout(value):
-#    return value == "pe", 30, "/percent"
+@APP.callback(
+    Output("map", "center"),
+    Output("latInput", "min"),
+    Output("latInput", "max"),
+    Output("lngInput", "min"),
+    Output("lngInput", "max"),
+    Output("latLab", "children"),
+    Output("lonLab", "children"),
+    Input("submitLatLng","n_clicks"),
+)
+def initialize(toto):
```

Comment:
It was I who suggested that we find the center coordinates in a callback rather than in the layout, so that we could get it from the dataset instead of duplicating the information in the config file. Here we see a disadvantage of doing it that way: a callback must have inputs, so you were forced to declare an arbitrary one even though this function doesn't really depend on it. Now this callback will be called unnecessarily each time the user submits the form.

I'm thinking of a couple of alternative setups, but it would be easier to implement them than to explain them. This is fine for now, and I will try alternatives as part of making a multi-page ENACTS maproom setup.